### ISSUE TITLE
Missing 

### ISSUE THREAD

**Issue by xambroz** [volatilityfoundation/volatility#230]
```
There are missing __init__.py files in contrib directory in order to be used as plugin directory in read-only environment.
Please can you add the files?

          touch volatility/contrib/plugins/__init__.py
          touch volatility/contrib/plugins/malware/__init__.py

Thank you
Michal Ambroz
```

**Comment by gleeda** [comment]
@ikelos i can't remember, but was there a reason why we left out the `__init__.py` files in those folders ?  

That being said, you can also just point directly to the folder you want to use the plugin from, like: 

`--plugins=contrib/plugins/malware/`

**Comment by ikelos** [comment]
Errrrr, not that I can recall?  Plugin libraries do rewrite the __init__.py files to exist in any subdirectories to be usable.  Perhaps we left them out because the contrib folder isn't really maintained as much as core?  Either way, there shouldn't be a problem touching the files and committing them back in... 5:)

**Comment by gleeda** [comment]
Thanks!  committed now :-)

**Comment by gleeda** [comment]
fixed in 649424302bac174f432ba1ebca9f4885d54ede82

**Comment by xambroz** [comment]
Thank you
